### DISCUSSION TITLE
Can't open in macos

### DISCUSSION THREAD

**Discussion by ahammednibras8** [KRTirtho/spotube#1131] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

“spotube” can’t be opened because Apple cannot check it for malicious software.

### Expected Behavior

just work

### Steps to reproduce

nothing

### Operating System

Mac OS

### Installation source

Website (spotube.netlify.app) or (spotube.krtirtho.dev)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by chanthinh** [comment]
@ahammed-nibras,
See this one: https://github.com/KRTirtho/spotube/issues/1086
